app/views/layouts/application.html.erb

```diff
@@ -2,7 +2,6 @@
 <html lang="en">
   <head>
     <meta name="viewport" content="width=device-width,initial-scale=1">
-    <meta name="turbo-cache-control" content="no-cache">
```

Comment:
I noticed a very tiny blink of the right list of videos.
The root cause is that for now, this list is a random selection of 5 talks so it changes for each visit.
https://github.com/adrienpoly/rubyvideo/assets/7847244/9a8fcd89-4478-4793-856f-92e2bb173fbb

I see two solutions:
1. add a Stimulus controller and remove this list content on disconnect
2. change the way we generate this list to have the same result for every visit

Reply:
Yes, this is an odd user experience. Another option would be to disable previews for that view. I often do this by setting `@turbo_cache_control = :no_preview` in the template.

Reply:
will change that perfect solution